### PR TITLE
[PAY-924][Desktop] DMs: Blocking/Unblocking Wire Up/UI

### PR DESCRIPTION
**Pull request by rickyrombo**
### Description

- Adds blocking/unblocking actions to store, including getting the blockee list.
- Wires it up to the UI dropdowns

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

There's no handler for errors right now, nor any real feedback that what you did is successful. I think maybe we want a Toast for this?

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

File: packages/web/src/components/stat-banner/StatBanner.tsx [hunk 1]
```diff
@@ -146,13 +153,17 @@ export const StatBanner = (props: StatsBannerProps) => {
...
-                    text: messages.unblockMessages,
-                    onClick: () => {
-                      // TODO
-                    },
-                    icon: <IconUnblockMessages />
-                  }
+                  isBlocked
+                    ? {
+                        text: messages.unblockMessages,
+                        onClick: onUnblock!,
```
Comment (dharit-tan) [inline packages/web/src/components/stat-banner/StatBanner.tsx:159]: do we need `!`? Can the prop be not optional?

**Review by dharit-tan — APPROVED** [review]

**Comment by dharit-tan** [comment]
typecheck!

File: packages/web/src/components/stat-banner/StatBanner.tsx (hunk 1 above)
Comment (rickyrombo) [reply packages/web/src/components/stat-banner/StatBanner.tsx:159]: StatBanner can be empty, eg on track pages

**Comment by rickyrombo** [comment]
> typecheck!

dep on #3160

**Pull request merged**